src/Tribe/Capabilities.php

```diff
@@ -1,9 +1,40 @@
 <?php
 
 class Tribe__Events__Capabilities {
-	public $set_initial_caps = false;
-	private $cap_aliases = array(
-		'editor' => array( // full permissions to a post type
+
+	/**
+	 * The transient key for delayed capabilities updates.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected $key_delayed_capabilities_update = '_tribe_events_delayed_capabilities_update';
+
+	/**
+	 * An array of roles to update capabilities.
+	 *
+	 * @since TBD
+	 *
+	 * @var array
+	 */
+	protected $roles = [
+		'administrator',
+		'editor',
+		'author',
+		'contributor',
+		'subscriber'
+	];
+
+	/**
+	 * An array of capabilities aliases by role.
+	 *
+	 * @since TBD
+	 *
+	 * @var array
+	 */
+	private $cap_aliases = [
+		'editor' => [ // full permissions to a post type
```

Comment:
* :warning: Array double arrow not aligned correctly; expected 6 space(s) between "'editor'" and double arrow, but found 1.
* :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: ffe2afc7ff876f8a9ff645fcb78f6250</sub>